### DISCUSSION TITLE
How to use headers for my blog post? Or use a different .md structure alltogether?

### DISCUSSION THREAD

**Discussion by Najib90** [mearashadowfax/ScrewFast#163] [Q&A] [answered]
I understand that search engines prefer headers over plain text for crawling. What’s the best way to incorporate headers into longer blog posts for improved SEO? Or is my understanding wrong?

**Comment by mearashadowfax** [comment]
To optimize your blog posts for SEO, you can use a structure from the insights content collection. Here's a basic structure you can follow:

```
# Main Title of the Blog Post (h1)

## Introduction (h2)

### Sub-topic 1 (h3)

#### Further Detail on Sub-topic 1 (h4)
```

**Reply by Najib90** [reply]
Could you clarify whether the [...slug].astro file from insights is intended to be used for creating blog posts, rather than the file currently employed by the blog? Additionally, will this approach provide performance comparable to the existing solution, or would you suggest an alternative library for managing blog posts?

Currently, using pound signs alone is not effective

**Reply by mearashadowfax** [reply, marked as answer]
You use the rest parameter ([...path]) to match file paths, so the one in the blog/ directory matches the blog content collection, and the one in the insights/ directory matches the insights content collection. The main difference lies in the structure of the Markdown files in these collections. If you have sub-headings and are concerned about SEO, you might want to adopt the structure from the insights collection and make some adjustments to [...slug].astro in the blog directory. An alternative way to manage blog posts is to use a CMS.

**Reply by Najib90** [reply]
Thank you, I did what you suggested and borrowed from the insights to create a new layout for the blog.
